### PR TITLE
feat: spring improvements

### PR DESCRIPTION
**Pull request by jhagestedt**

### CODE REVIEW

**Comment by jhagestedt** [comment]
Taken from https://github.com/corona-warn-app/cwa-verification-server/pull/59

File: src/main/java/app/coronawarn/verification/service/TanService.java [hunk 1]
```diff
@@ -182,12 +172,12 @@ private VerificationTan persistTan(String tan, TanType tanType, TanSourceOfTrust
...
   public String generateTeleTan() {
     return IntStream.range(0, TELE_TAN_LENGTH)
-        .mapToObj(i -> TELE_TAN_ALLOWED_CHARS.charAt(Holder.NUMBER_GENERATOR.nextInt(TELE_TAN_ALLOWED_CHARS.length())))
-        .collect(Collector.of(
-            StringBuilder::new,
-            StringBuilder::append,
-            StringBuilder::append,
-            StringBuilder::toString));
+      .mapToObj(i -> TELE_TAN_ALLOWED_CHARS.charAt(Holder.NUMBER_GENERATOR.nextInt(TELE_TAN_ALLOWED_CHARS.length())))
+      .collect(Collector.of(
```
Comment (odrotbohm) [inline src/main/java/app/coronawarn/verification/service/TanService.java:176]: `Collectors.joining()`?

Comment (jhagestedt) [reply src/main/java/app/coronawarn/verification/service/TanService.java:176]: well i think current way also fits needs.

Comment (odrotbohm) [reply src/main/java/app/coronawarn/verification/service/TanService.java:176]: I don't dispute that. I just thought it's a bit laborious to create a custom `Collector` for something that there's a prepared one available. Of course, your call.

**Pull request merged**